rps_game.js

```diff
@@ -14,8 +14,8 @@ function isDraw(userChoise, computerChoise) {
 
 function humanWins(userChoise, computerChoise) {
   return userChoise === 'rock' && computerChoise === 'scissors' ||
-         userChoise === 'paper' && computerChoise === 'rock' ||
-         userChoise === 'scissors' && computerChoise === 'paper';
+  userChoise === 'paper' && computerChoise === 'rock' ||
+  userChoise === 'scissors' && computerChoise === 'paper';
```

Comment:
I think it was better before :joy: (with spaces to align them)
no need to change it though :+1: